src/include/sof/low_latency_schedule.h

```diff
@@ -31,8 +31,8 @@
  * interrupt source. It should execute quickly and must not sleep or wait.
  */
 
-#ifndef __INCLUDE_WORK__
-#define __INCLUDE_WORK__
+#ifndef __INCLUDE_SOF_LOW_LATENCY_SCHEDULE_H__
```

Comment:
This work.h header should be merged into schedule.h as well. We need to share task/work structures in order to unify our APIs.

Reply:
@lgirdwood 
I've done structure of function pointers for each specific scheduler (and export it by "extern" from specific scheduler header file).
Scheduler specific headers are included in scheduler.c file to use this specific structs.
I think generic schedule.h file shouldn't include specific scheduler headers, because there arise circular dependency between headers.